### PR TITLE
Max limit made 99999 and release version updated

### PR DESCRIPTION
**Pull request by sravan1000**
## Description

Removed Max limit 50 and made 99999

## Related Issues

Issue: #XXX

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/query/queryWithCursor.spec.ts [hunk 1]
```diff
@@ -60,17 +60,6 @@ describe('queryWithCursor', () => {
     ).rejects.toThrowError('Partition key condition can only be a string');
   });
 
-  it('should throw an error when max limit is exceeded', async () => {
```
Comment (himanksahni) [inline src/query/queryWithCursor.spec.ts:63]: Shouldn't we test for when the limit is **100000** ?

**Review by mdsiddiq-till — DISMISSED** [review]
Looks good.
nit: Use constant for max limit

File: src/query/queryWithCursor.spec.ts (hunk 1 above)
Comment (sravan1000) [reply src/query/queryWithCursor.spec.ts:63]: if limit is 10000, then we should fetch 10000 records

**Comment by sravan1000** [comment]
@mdsiddiq-till Pr comments addressed

File: src/query/queryWithCursor.ts [hunk 2]
```diff
@@ -32,12 +32,7 @@ export async function queryWithCursor<T extends AnyObject>(
...
   const params = buildQueryTableParams(filter, partitionKeyName, sortKeyName);
 
-  if (params.Limit) {
-    if (params.Limit > MAX_LIMIT) {
-      throw new Error(`Maximum limit of ${MAX_LIMIT} can be applied`);
-    }
-  } else {
-    // apply a default limit
+  if (!params?.Limit) {
     params.Limit = MAX_LIMIT;
```
Comment (mdsiddiq-till) [inline src/query/queryWithCursor.ts:36]: Can we rename it to DEFAULT_LIMIT?

**Review by snehithv — APPROVED** [review]

**Comment by snehithv** [comment]
@sravan1000 one test is failing

**Comment by sravan1000** [comment]
@snehithv Tets fixed when re-run

File: src/query/queryWithCursor.ts (hunk 2 above)
Comment (sravan1000) [reply src/query/queryWithCursor.ts]: changed

**Review by mdsiddiq-till — APPROVED** [review]

File: src/query/queryWithCursor.ts [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { decrypt, encrypt } from '../utils';
 import { AnyObject, Filter, TableConfig } from '../types';
 import { buildQueryTableParams } from './queryBuilder';
 
-const MAX_LIMIT = 50;
+const DEFAULT_LIMIT = 99999;
```
Comment (NalinSajwan) [inline src/query/queryWithCursor.ts:6]: Why such a big number `99999` ? 

Should loading `100` records by default not be sufficient ? User already has property to override it if they want.

Comment (sravan1000) [reply src/query/queryWithCursor.ts:6]: No, in progress orders may contain any count and sure it will be less than `99999`

**Pull request merged**